kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/Application.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.good.patterns.food2door;
+
+import com.kodilla.good.patterns.food2door.database.OrdersRepository;
+import com.kodilla.good.patterns.food2door.infoRetrieving.*;
+import com.kodilla.good.patterns.food2door.notification.MailInfo;
+import com.kodilla.good.patterns.food2door.order.Order;
+
+import com.kodilla.good.patterns.food2door.orderProcess.OrderProcessor;
+
+public class Application {
+
+    public static void main(String[] args) {
+
+        OrderRetriever orderRetriever = new OrderRetriever();
+        Order order = orderRetriever.retrieve();
+
+        OrderProcessor orderProcessor = new OrderProcessor(new MailInfo(), new OrdersRepository());
+        orderProcessor.execute(order);
+        order.getSupplier().process(order);
```

Comment:
1. Produkty z zamówienia `order` powinny być w jakiś sposób pogrupowane po dostawcach i dla każdego z nich (dostawców) powinno być przeprocesowane zamówienie produktów tylko od tego dostawcy. 
2. Wszystko to powinno dziać się wewnątrz `orderProcessor`. Ta klasa powinna przykrywać całą złożoność związaną z różnorodnymi systemami dla dostawców, występującymi pod spodem.
3. Do rozważenia też, jak ma się zachować system w przypadku problemu z zamówieniem od jednego lub więcej dostawców. Czy wtedy chcesz anulować całe zamówienie, czy złożyć zamówienie niekompletne? To powinno mieć odzwierciedlenie w obiekcie zwracanym z `orderProcessor`-a.